frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -325,22 +325,45 @@
                 parameters.url = 'hosts/challenge_host_teams/' + hostTeamId + '/invite';
                 parameters.method = 'POST';
                 parameters.data = {
-                    "email": result
+                    "email": result,
+                    "url": $location.absUrl(),
                 };
                 parameters.token = userKey;
                 parameters.callback = {
                     onSuccess: function() {
                         $rootScope.notify("success", parameters.data.email + " has been invited successfully");
                     },
-                    onError: function() {
-                        $rootScope.notify("error", "Couldn't invite " + parameters.data.email + ". Please try again.");
+                    onError: function(response) {
+                        $rootScope.notify("error", "Couldn't invite. " + response.data.error + " Please try again.");
                     }
                 };
 
                 utilities.sendRequest(parameters);
             });
         };
 
+        if ($stateParams.email_hash){
```

Comment:
__Bug__ : this `if(...)` is also executing in other routes that uses same controller (i.e. [http://127.0.0.1:8888/web/challenge-host-teams](http://127.0.0.1:8888/web/challenge-host-teams))

### How to solve this:
1. write a `vm.custom_function()` and move all the code that is inside `if(){...}`
```
vm.custom_function = function() {
        var team_hash = $stateParams.team_hash;
        .....
        .....
        .....
       // All the code that is inside if statement
     
}
```
2. Change route configuration and call `custom_function` from inline template
```
 var host_invitation = {
        name: "web.host-invitation",
        parent: "web",
        url: "/challenge-host-invitation/:team_hash/:email_hash",
        controller: 'ChallengeHostTeamsCtrl',
        controllerAs: 'challengeHostTeams',
        template: '<span ng-init="challengeHostTeams.custom_function()"><span>',
        title: 'Host Invitation',
        authenticate: true
 };
```
Done!!